app/errorHandlers.js

```diff
@@ -1,15 +1,15 @@
 exports.handle400s = (err, req, res, next) => {
   console.log(err);
-  if (err.code === "22P02" || err.code === "23502") {
-    res.status(400).send({ msg: "Bad request" });
-  } else if (err.code === "23503") {
-    res.status(404).send({ msg: "Not found" });
-  } else if (err.status === 400) {
-    res.status(err.status).send({ msg: err.msg });
-  }
+  // if (err.code === "22P02" || err.code === "23502") {
```

Comment:
is this meant to be commented out?

Reply:
Got your point, will use them when I get to the errors